chapter-customizing.asciidoc

```diff
@@ -1289,16 +1290,16 @@ show that this parameter declares an expression:
 
 .Plugin Parameter Expressions
 ----
-testFailureIgnore  Set this to true to ignore a failure during \
-testing. Its use is NOT RECOMMENDED, but quite \
-convenient on occasion.
+testFailureIgnore  Set this to true to ignore a failure during 
+                   testing. Its use is NOT RECOMMENDED, but quite 
+                   convenient on occasion.
 
 * Type: boolean
 * Required: No
-* Expression: ${maven.test.failure.ignore}
+* User Property: maven.test.failure.ignore
```

Comment:
The current Surefire documentation is calling these "user properties" and has no reference to an "expression" that I could see.